Dockerfile

```diff
@@ -3,14 +3,25 @@ FROM docker.io/centos:centos7
 RUN yum install -y python-requests && \
     curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/tripleo_repos/main.py | python - -b stein current-tripleo && \
```

Comment:
We have to stop doing this.

Reply:
can't we keep doing it for metal3?
downstream we can access stable packages anyway, if they're not available in rhel then we probably have to find an alternative, but for now we have access to osp pkgs

Reply:
isn't the goal here to stop using RDO/OSP packages?

Reply:
Kind of is the goal. Where can we find crudini ?

Reply:
the goal is to stop using anything coming from OSP/RDO, but for some packages if they're available in old version that are accessible and usable I don't see why not keeping doing that until we find an alternative

Reply:
also there are alternatives to crudini if we really can't find a way to use it downstream

Reply:
> Kind of is the goal. Where can we find crudini

crudini can be also installed from pip?

> for some packages if they're available in old version that are accessible and usable I don't see why not keeping doing that until we find an alternative

right, but it's pretty urgent

Reply:
crudini is available via pypi last time I checked.  https://pypi.org/project/crudini/

Reply:
@yazug  thanks! I will update the list :)